ui/pages/confirm-send-ether/confirm-send-ether.container.js

```diff
@@ -1,7 +1,7 @@
 import { connect } from 'react-redux';
```

Comment:
At some point - not in this PR - we should consider renaming these files/components/containers towards more chain-agnostic language